Gemfile.lock

```diff
@@ -162,6 +160,8 @@ GEM
 
 PLATFORMS
   arm64-darwin
+  x64-mingw-ucrt
```

Comment:
"UCRT64: The Universal C Runtime (UCRT) is used by recent versions of Microsoft Visual Studio. It ships by default with Windows 10."